src/History/NewKeyword.jsx

```diff
@@ -0,0 +1,11 @@
+import PropTypes from "prop-types";
+
+export default function NewKeyword({ Keyword }) {
```

Comment:
P1 : 매개변수가 string 이니까 소문자로 시작하면 좋을 것 같아요
```suggestion
export default function NewKeyword({ keyword }) {
```

Reply:
의견주셔서 감사합니다:) 수정완료하였습니다! 
<img width="1019" alt="image" src="https://github.com/user-attachments/assets/8a84e2e0-3fa0-4f5f-bc13-b66308b9cefc" />